.travis/package_management/configure_rpm_lxc_environment.py

```diff
@@ -49,7 +49,7 @@
 print("2.1 Preparing repo on LXC container")
 common.prepare_repo(container)
 
-common.run_command(container, ["wget", "-T", "15", "-O", "/home/%s/.install-required-packages.sh" % (os.environ['BUILDER_NAME']), "https://raw.githubusercontent.com/netdata/netdata-demo-site/master/install-required-packages.sh"])
+common.run_command(container, ["wget", "-T", "15", "-O", "/home/%s/.install-required-packages.sh" % (os.environ['BUILDER_NAME']), "https://raw.githubusercontent.com/netdata/netdata/master/packageing/installer/install-required-packages.sh"])
```

Comment:
I think there is a typo here: `packageing` instead of `packaging`

Reply:
Indeed it is a typo. I'll have a fix pushed shortly.